docs/product/custom-dashboards.md

```diff
@@ -5,74 +5,92 @@ sidebar_position: 6
-3. Create 2 queries for Network Request, the first filtered for Status Codes in the range 200-299, and the second in the range 400-599.
-4. In the Formula box, add the formula `A/(A+B) * 100` to get a percentage.\
+1. Create 2 queries for Network Request, the first filtered for Status Codes in the range 200-299, and the second in the range 400-599.\
+    a. You can also add a group by here. For this example, we group by App Version. See more on grouping functionality below. 
+    b. Formulas and multiple queries are only supported on line charts currently. 
+2. In the Formula box, add the formula `A / (A+B) * 100` to get a successful network request percentage by App Version.\
     a. The formula field accepts the operators: `+, -, *, /`.\
     b. Order of operations follows standard PEMDAS.
-5. Hide the queries above by clicking on the eye symbols to just show the formula, which is your rate.
+3. Hide the queries by clicking on the eye symbols to just show the formula, which is your successful network request percentage.
 
-<img src={require('@site/static/images/combined_widget_multiquery_formula.png').default} alt="Multiple queries and formula" />
-
-Note: Queries in Combined type charts do not support group-bys.
+<img src={require('@site/static/images/custom_dashboards/Formula_Timeseries_Example.png').default} style={{ width: '75%', height: '75%' }} alt="Multiple queries and formula" />
 
 ### Multiple Combined Time Series without a Formula
 
 Since the Formula field is optional, you can visualize up to 10 time series on the same chart with the steps above. Use the eye symbols to show or hide each and leave the Formula field blank.
 
-## Table of Issues
+### Grouping with Combined Time Series
+
+In the broadest sense, the group-by's for each query must have some overlapping set. For example, if you were to set one query to "Crash Count" grouped by "App Version", and the second query to "Session Count", grouped by "Build", then perform some operation on them, this would be an illegal formula.
 
-Our Issues Widget, lets you specify how to list a table of [Issues](/product/issue-monitoring-and-work-flow) .  You can filter for certain Issue types, add filters to limit app-versions, or select just Issues [tagged to your team.](/product/tagging)
+Additionally, order of operations matters. Consider a situation where you create three queries:
+- `A` grouped by `[app_version, os_version]` 
+- `B` grouped by `[app_version]` 
+- `C` grouped by `[os_version]` 
+
+Then you were to combine them in the formula field:
+
+:white_check_mark: `(A + B) + C`: `A` and `B` share a grouping dimension and can combine, then combine with `C`.\
+:x: `A + (B + C)`: `B` and `C` share now group-by dimension and therefore fail. 
+
+<img src={require('@site/static/images/custom_dashboards/Grouping_Error.png').default} style={{ width: '75%', height: '75%' }} alt="Example of Grouping Error" />
+
```

Comment:
We should also mention that if we join queries A and B and they have no common labels we will get zero results. So if A has app_version={1.0, 2.0} and app_version={3.0} then they will get zero results.